source/NPC.cpp

```diff
@@ -131,6 +137,61 @@ void NPC::Load(const DataNode &node)
 			else
 				location.Load(child);
 		}
+		else if(child.Token(0) == "waypoint")
+		{
+			// Waypoints can be added one-by-one (e.g. if from LocationFilters) or
+			// from multiple "waypoint" nodes. If no system or filter to pick a system is
+			// passed to the node, the mission's destination system will be used.
+			if(!child.HasChildren())
+			{
+				// Given "waypoint" or "waypoint <system 1> .... <system N>"
```

Comment:
I'm going to leave this as-is, because it's technically invalid grammar with American English to have the period outside the quotes, and obviously it wouldn't be a good idea to put them inside.